### DISCUSSION TITLE
Created a Reinstatement Request and I am no getting any response from GitHub

### DISCUSSION THREAD

**Discussion by souraavsen** [community/community#120727] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My account is flagged. My account was hacked before but now its been recovered. I need my account back from being marked as flagged because my work kinda depends on this. I can not do any Git-related actions. And I saw that most people get responses in more than 3 weeks unfortunately so I am scared I won't get it as soon as I need it. This is ridiculous. My username is @souraavsen1

Anyone know any action I can take so they respond quickly?
Thanks in advance.

**Comment by david-wei-01001** [comment]
Having your GitHub account flagged can indeed be distressing, especially if it affects your work. Here's what you can do to try and expedite the process:

1. **Follow Up on Your Request**: If it’s been a few days since you filed the reinstatement request and haven't heard back, you can send a polite follow-up email to ask for an update on the status of your appeal. Make sure to reference any ticket numbers or case IDs that were provided when you initially contacted them.

2. **Provide Detailed Information**: In your communication with GitHub, provide all relevant information that might help them investigate and resolve the issue quicker. This includes details about how the account was compromised and the steps you've taken to secure it.

3. **Check Your Email, Including Spam**: Sometimes, responses from support teams can end up in the spam or junk folder. Make sure you're checking all your email folders.

4. **Reach Out on Social Media**: Some users have reported success getting attention by reaching out to GitHub via social media, such as Twitter. If you choose to go this route, keep the message concise, polite, and professional.

5. **Utilize Community Support Forums**: GitHub has a community forum where you can post about your issue. Sometimes community managers and other experienced users offer advice that can be helpful.

6. **Ensure Account Security**: While waiting for a response, make sure your account is secure. Use a strong password, enable two-factor authentication, and review your account for any unknown third-party applications or tokens that should be revoked.

7. **Be Patient But Persistent**: While it can be frustrating to wait, especially when your work depends on it, understand that support teams often handle a large volume of requests. They typically process these requests in the order they are received. However, it’s okay to be persistent and follow up at regular intervals.

8. **Prepare Documentation**: Have any relevant documentation at hand to prove your identity and the legitimacy of your request. This can help speed up the verification process.

9. **Plan for Contingencies**: If GitHub is crucial to your workflow, consider creating a secondary backup account (if GitHub's terms allow it) and setting up necessary configurations to continue your work with minimal disruption in case such issues arise in the future.

Remember to avoid sending too many messages in a short time, as this can be counterproductive. If there is a significant delay, or if your work is being severely impacted, you might also try reaching out to GitHub Support via any other official contact methods they have listed. Keep all communication respectful and include all pertinent information to aid in the resolution of your issue.

**Reply by souraavsen** [reply]
Thank you @david-wei-01001. Yes its really furstating that lot of things are stucked. its been 1 month almost, I have created the ticket. But still no response. Thats why share here if there any other ways.

**Comment by ghost** [comment]
I have already submitted the ticket for around two months. Still no response.

**Comment by OverclockedD2** [comment]
Same problem here, been waiting for over 60 days now. They just keep ignoring, no communication at all. I'm honestly considering just making a new account at this point, this is pretty damn annoying.
